packages/core/src/item/basic_item.js

```diff
@@ -1,7 +1,10 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
 import React from "react";
 
-const BasicItem = ({ label = "" }) => {
-  return <span>{label}</span>;
+const BasicItem = props => {
+  const { label = "", getStyles } = props;
```

Comment:
This can be done in the method signature.

This whole thing can look like this:
```
const BasicItem =({ label = "", getStyles }) => 
     <span css={getStyles("selectedItem", props)}>{label}</span>;
```

Obviously this is just syntax preference, not critical in any way.

Reply:
I'm using props when calling getStyles.
I tried doing it the way you suggested and got an error that "props is not defined" (which is true ;) )

Reply:
({ label = "", getStyles, ...props })


Anyway, not critical at all.